git_hg_sync/application.py

```diff
@@ -56,10 +65,32 @@ def _handle_push_event(self, push_event: Push) -> None:
 
         for destination, operations in operations_by_destination.items():
             try:
-                synchronizer.sync(destination, operations)
+                retry(
+                    "executing sync operations",
+                    # Python gotcha: when creating closure in a loop, any loop
+                    # variable used in the closure takes the last value of the variable,
+                    # at the end of the loop [0].
+                    #
+                    # We can use functools.partial to early-bind those parameters into
+                    # a Callable.
+                    #
+                    # [0] https://docs.python-guide.org/writing/gotchas/#late-binding-closures
+                    partial(synchronizer.sync, destination, operations),
+                    tries=3,
+                    delay=5,
```

Comment:
This is the only call to `retry` that is also wrapped in a `try` block. Asking in case this was left by mistake, since exceptions are already caught and logged in `retry`.